src/components/link/link.tsx

```diff
@@ -0,0 +1,27 @@
+import { FC } from 'react';
+import cx from 'classnames';
+import CoreLink from '../../helpers/components/Link/Link';
+import styles from './link.module.scss';
+import { Size } from './LinkConstants';
+
+type LinkProps = {
+  className: string;
+  children: string;
+  href: string;
+  size?: Size;
+  withoutSpacing?: boolean;
+};
+
+const Link: FC<LinkProps> = ({ children, href, size = Size.MEDIUM, withoutSpacing = false, className }) => (
+  <CoreLink
+    text={children}
+    href={href}
+    className={cx(styles.compsLink, className, {
+      [styles.small]: size === Size.SMALL,
+      [styles.medium]: size === Size.MEDIUM,
+      [styles.withSpacing]: !withoutSpacing,
+    })}
+  />
+);
+
+export default Link;
```

Comment:
```suggestion
export default withStaticProps(Link, { sizes: Size } );
```

So we'd be able to use it like this e.g. in other packages
`<Link size={Link.sizes.MEDIUM}`

Same as we do in `Tip`